# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/ipc/CompositorBridgeParent.h
# Commit: fbf55a44d7fb
# Full Hash: fbf55a44d7fb72908eeb5d041b70ce2c503a2532
# Author: Anny Gakhokidze <agakhokidze@mozilla.com>
# Date: 2020-04-15 04:09:17
# Regressor Bug: 1594529
# File Overlap Count: 1
# Description:
#   Bug 1594529 - Create LoadInfo for subdocuments directly in parent process with DocumentChannel. r=mattwoodrow,nika
#   
#   Currently, with Fission enabled we are not able to create a proper LoadInfo
#   object when doing a subdocument load because we do not have access to a loading
#   context if the load is happening inside of an OOP frame. To solve this problem,
# ==============================================================================

diff -r 4ba9a230586a -r fbf55a44d7fb gfx/layers/ipc/CompositorBridgeParent.h
--- a/gfx/layers/ipc/CompositorBridgeParent.h	Mon Apr 13 22:41:37 2020 +0000
+++ b/gfx/layers/ipc/CompositorBridgeParent.h	Mon Apr 13 23:00:43 2020 +0000
@@ -53,9 +53,9 @@
 namespace mozilla {
 
 class CancelableRunnable;
+class HostWebGLCommandSink;
 
 namespace dom {
-class HostWebGLCommandSink;
 class WebGLParent;
 }  // namespace dom
 